msal/src/main/java/com/microsoft/identity/client/IMultipleAccountPublicClientApplication.java

```diff
@@ -42,7 +45,13 @@
      *
      * @param callback The callback to notify once this action has finished.
      */
-    void getAccounts(@NonNull final PublicClientApplication.LoadAccountCallback callback);
+    void getAccounts(@NonNull final LoadAccountsCallback callback);
+
+    /**
+     * Returns a List of {@link IAccount} objects for which this application has RefreshTokens.
+     */
+    @WorkerThread
+    List<IAccount> getAccounts() throws InterruptedException, MsalException;
```

Comment:
Shouldn't we return only MsalException?

Reply:
I think in the case of sync calls to call InterruptedException... sort of a reminder that you're making a synchronous background call that could be interrupted; however very open wrapping if we agree that's appropriate.  I'm assuming MsalClintException of type InterruptedException.